### ISSUE TITLE
Error for subcommands

### ISSUE THREAD

**Issue by Bearzooka** [univ-of-utah-marriott-library-apple/jctl#53]
Hey there,

I shared this in Slack, but I figured it would be better to have it here:

After running subcommands I get an error that says
```
Traceback (most recent call last):
  File "/Users/XXXX/.pyenv/versions/3.9.10/bin/jctl", line 593, in <module>
    main(sys.argv[1:])
  File "/Users/XXXX/.pyenv/versions/3.9.10/bin/jctl", line 371, in main
    print("Server: " + api.url)
AttributeError: 'API' object has no attribute 'url'
```

I made the subcommand work by manually editing line 371 in `jctl` and removing the call to `api.url` but I fear there might be something bigger in the background, so I guess it's worth checking.

Thanks!

**Comment by magnusviri** [comment]
@Bearzooka this is just a minor error. I've fixed it locally on my hard disk and will push it to the repo and release a fix soon.

**Comment by magnusviri** [comment]
I just fixed this with a new release, 1.1.18.
